sqlserver-persistence/src/main/resources/db/migration_sqlserver/V1__initial_schema.sql

```diff
@@ -0,0 +1,275 @@
+-- --------------------------------------------------------------------------------------------------------------
+-- Migration script V1 for SQLSERVER
+-- Table and index names are taken from MySQL
+-- --------------------------------------------------------------------------------------------------------------
+GO
+
+-- --------------------------------------------------------------------------------------------------------------
+-- TAGLES FOR METADATA DAO
+-- --------------------------------------------------------------------------------------------------------------
+
+-- meta_event_handler
+CREATE TABLE [dbo].[meta_event_handler] (
+    id INT PRIMARY KEY IDENTITY(1, 1),
```

Comment:
Recently (#2028) we removed the "id" column in all conductor tables (mysql/postges) since the id column is not used. I suggest we do the same here for consistency.

Reply:
I removed `id` from all the tables except `queue_message` because I need the `id`  column there.
https://github.com/TwoUnderscorez/conductor/commit/44bcafaac894648992c2e16e754a252d35d8e79b